src/libraries/Common/src/System/Security/Cryptography/AesGcm.Windows.cs

```diff
@@ -4,43 +4,52 @@
 using System.Diagnostics.CodeAnalysis;
 using Internal.Cryptography;
 using Internal.NativeCrypto;
+using System.Runtime.InteropServices;
 
 namespace System.Security.Cryptography
 {
     public partial class AesGcm
     {
         private SafeKeyHandle _keyHandle;
+        private static readonly KeySizes s_tagByteSizes = new KeySizes(12, 16, 1);
 
-        public static bool IsSupported => true;
-        public static KeySizes TagByteSizes { get; } = new KeySizes(12, 16, 1);
+#if NET || NETFRAMEWORK
+        public static partial bool IsSupported => true;
+#elif NETSTANDARD
+        public static partial bool IsSupported => RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
In the csproj file, the `AesGcm.Windows.cs` implementation is used for .NET Standard and .NET Framework. AesGcm's constructor checks `IsSupported` and will fail with PNSE if it is false.